src/plugins/shared_ux/public/components/exit_full_screen_button/exit_full_screen_button.component.tsx

```diff
@@ -48,49 +43,35 @@ export const ExitFullScreenButton = ({ onClick, className }: Props) => {
   const { euiTheme } = useEuiTheme();
   const { colors, size, border } = euiTheme;
 
-  const textCSS = css`
-    line-height: 1.2;
-    ${makeHighContrastColor(colors.mediumShade)(colors.fullShade)};
-  `;
-
-  const iconCSS = css`
-    ${makeHighContrastColor(colors.mediumShade)(colors.fullShade)};
-  `;
-
   const buttonCSS = css`
-    display: inline-block;
     padding: ${size.xs} ${size.s};
-    border: none;
-    background: none;
     background: ${colors.fullShade};
     border-radius: ${border.radius.small};
-    text-align: left;
-    height: 32px;
-    color: ${colors.lightShade};
-
-    &:hover {
-      background: ${colors.fullShade};
-      color: ${colors.emptyShade};
-    }
+    height: ${size.xl};
```

Comment:
nice catch, thank you!

Reply:
Cool to see the new theme tokens in use!